pom.xml

```diff
@@ -98,6 +98,18 @@
     </dependencies>
   </dependencyManagement>
   <dependencies>
+    <dependency>
+      <groupId>cglib</groupId>
+      <artifactId>cglib</artifactId>
+      <version>3.3.0</version>
+      <!-- This exclusion must remain as the group id changed. -->
+      <exclusions>
+        <exclusion>
+          <groupId>org.ow2.asm</groupId>
+          <artifactId>asm</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
I don't know what cglib is, but it has some pretty serious warnings in its [source repo](https://github.com/cglib/cglib) that give me strong "we shouldn't be doing this" vibes:

> IMPORTANT NOTE: cglib is unmaintained and does not work well (or possibly at all?) in newer JDKs, particularly JDK17+. If you need to support newer JDKs, we will accept well-tested well-thought-out patches... but you'll probably have better luck migrating to something like [ByteBuddy](https://bytebuddy.net/).

Reply:
As noted in the comments to the overall pull request

"Use the last cglib to avoid any potential issues (at least until apache commons team finally releases digester upgrade after 14 years) I expect this to be removed at that time."

Reply:
for what commons-digester3 is doing now which is active today even, see https://github.com/apache/commons-digester/pull/9